WTWSMS/common/landed_titles/mercenaries.txt

```diff
@@ -2565,4 +2564,80 @@ d_ghanan_band = {
 	strength_growth_per_century = 1.0
```

Comment:
> Concensus is the absence of leadership.

That quote has no relevance to this context.

> Imagine if everyone had to take up every bug fix or minor change with you. 

You make suggestions if you want to reinvent the wheel. And yes, I look through all changes everywhere. It is kind of presumed that you do that as an active developer.